src/ComScoreStreamingAnalytics.ts

```diff
@@ -127,6 +168,8 @@ export class ComScoreStreamingAnalytics {
 
   private playbackFinished = (event: PlayerEventBase) => {
     this.stopComScoreTracking();
+    this.streamingAnalytics = null;
+    new ComScoreStreamingAnalytics(this.player, this.metadata, this.configuration);
```

Comment:
Why initializing a new instance of `ComScoreStreamingAnalytics` without using the instance?